contracts/account-history-contract/src/action/sudo/update_account.rs

```diff
@@ -20,129 +19,110 @@ pub fn update_account(deps: DepsMut<ElysQuery>, env: Env) -> StdResult<Response<
     metadata = metadata.update_prices(&querier)?;
     METADATA.save(deps.storage, &metadata)?;
 
-    // update pagination
-    let mut pagination = PAGINATION.load(deps.storage)?;
-
-    let resp = querier.accounts(Some(pagination.clone())).map_err(|e| {
-        StdError::generic_err(format!(
-            "failed to get accounts with pagination {:?}: {}",
-            pagination, e
-        ))
-    })?;
-
-    pagination.update(resp.pagination.next_key);
-    PAGINATION.save(deps.storage, &pagination)?;
-
     let today = get_today(&env.block);
+    let user_address_queue: Vec<String> = USER_ADDRESS_QUEUE
```

Comment:
@politeWall can you also add a new query that will provide the current number of user addresses in the queue, we want to make sure that we process enough addresses per block (limit set by default to 1) to make up for all the addresses in a 24 hrs window

Reply:
yes query added [here](https://github.com/elys-network/bindings/pull/301/files#diff-fc69f4d4e3b05fb0eb6e7618fa7c1e13f6b8d8b4f19efdc7d93debe31a2cc8f9R166-R175)